### PR TITLE
Windows runtime events

### PR DESCRIPTION
**Pull request by TautvydasZilys**

### CODE REVIEW

File: mcs/class/corlib/System.Runtime.InteropServices.WindowsRuntime/UnsafeNativeMethods.cs [hunk 1]
```diff
@@ -1,10 +1,10 @@
-//
-// ReadOnlyArrayAttribute.cs
```
Comment (TautvydasZilys) [inline mcs/class/corlib/System.Runtime.InteropServices.WindowsRuntime/UnsafeNativeMethods.cs]: This wasn't supposed to be a rename: I have no idea how it ended up that way.

**Comment by TautvydasZilys** [comment]
My PR adds 9 new internal calls to mscorlib - am I supposed to add anything on mono native side (even if their implementation is empty)?

**Comment by joshpeterson** [comment]
I don't know if anything needs to be done for the new icalls. Maybe @joncham will have a better idea. You could try making a builds.zip and running some tests on the Mono upgrade branch of Unity to see if the new icalls cause an issue.

**Comment by TautvydasZilys** [comment]
Those new icalls will definitely not get called on Mono - the only way to get to them is to support windows runtime.  I was more thinking about the requirements of being accepted to upstream.

**Comment by joncham** [comment]
I think for upstream you could consider adding those icalls on the native side and raising not implemented exceptions. I guess we'd need feedback from upstream to know. Right now they will raise a MissingMethodException if they get called I think.

**Comment by TautvydasZilys** [comment]
Yeah, we need their input. How about I merge this now, get it to il2cpp-dependencies so it doesn't block me and concurrently open a PR to upstream to see what they have to say about it?

**Comment by joshpeterson** [comment]
That sounds fine to me.

**Comment by joncham** [comment]
LGTM

**Pull request merged**